tiatoolbox/tools/patchextraction.py

```diff
@@ -142,24 +146,31 @@ def __init__(
                 self.mask = None
             else:
                 self.mask = self.wsi.tissue_mask(
-                    method=input_mask, resolution=1.25, units="power"
+                    method=input_mask,
+                    resolution=1.25,
+                    units="power",
                 )
         elif isinstance(input_mask, wsireader.VirtualWSIReader):
             self.mask = input_mask
         else:
             self.mask = wsireader.VirtualWSIReader(
-                input_mask, info=self.wsi.info, mode="bool"
+                input_mask,
+                info=self.wsi.info,
+                mode="bool",
             )
         self.within_bound = within_bound
 
     def __iter__(self):
+        """Returns an iterator for the given object."""
         self.n = 0
         return self
 
-    def __len__(self):
+    def __len__(self) -> int:
+        """Returns the length of the instance attributes."""
```

Comment:
```suggestion
        """Return the length of the instance attributes."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/2db02db7a6949e400ee54fe03a5dfa0021412f44